tools/serverpod_cli/lib/src/analyzer/models/model_parser/model_parser.dart

```diff
@@ -152,10 +141,21 @@ class ModelParser {
             '/// the id will be null.',
           ],
         ),
-        ...fields,
-      ];
+      );
     }
 
+    var fieldsNode = documentContents.nodes[Keyword.fields];
+    if (fieldsNode is! YamlMap) return fields;
+
+    fields.addAll(fieldsNode.nodes.entries.expand((fieldNode) {
+      return _parseModelFieldDefinition(
+        fieldNode,
+        docsExtractor,
+        extraClasses,
+        serverOnlyClass,
+      );
+    }).toList());
```

Comment:
Style question: Do you generally prefer the full function syntax in this project over the shorthand arrow?

Reply:
Not really, but we use function syntax if the body is multiple lines long.
This usually helps with code readability.